rules/windows/execution_register_server_program_connecting_to_the_internet.toml

```
@@ -7,7 +7,7 @@ updated_date = "2020/08/03"
 [rule]
 author = ["Elastic"]
 description = """
-Identifies the native Windows tools regsvr32.exe and regsvr64.exe making a network connection. This may be indicative of
+Identifies the native Windows tools regsvr32.exe regsvr64.exe or RegAsm.exe making a network connection. This may be indicative of
```

```suggestion
Identifies the native Windows tools regsvr32.exe, regsvr64.exe, or RegAsm.exe making a network connection. This may be indicative of
```

May need to re-lint after this